setup.cfg

```diff
@@ -57,6 +57,7 @@ dev =
     pytest-faulthandler
     pytest-rerunfailures
     pytest-timeout
+    pyright[nodejs]
```

Comment:
Why the `[nodejs]` version?

Reply:
Strangely enough, `pyright` is written in Javascript, I suppose because it's closely tied to pylance and VSCode.
Adding the `[nodejs]` feature just makes it install nodejs automatically using a wheel and is recommended (see https://pypi.org/project/pyright/)

> It's highly recommended to install pyright with the nodejs extra which uses [nodejs-wheel](https://pypi.org/project/nodejs-wheel-binaries/) to download Node.js binaries as it is more reliable than the default [nodeenv](https://pypi.org/project/nodeenv/) solution.